src/types/Work.ts

```diff
@@ -2,7 +2,7 @@ import { Crew } from './Crew';
 
 export interface Work {
     id: string;
-    title: string;
+    title: string | undefined;
```

Comment:
```suggestion
    title: string;
```

necessarily fetch title always because title is shown as various places in UI and I don't want a stale title to be somewhere in the app